src/jvmTest/java/org/snakeyaml/engine/v2/scanner/FuzzYAMLRead50431Test.java

```diff
@@ -53,7 +51,7 @@ public void testIncompleteValue() {
     } catch (ScannerException e) {
       assertEquals(
           "found unknown escape character E30EEE",
-          e.getMessage().lines().findFirst().orElse(null)
+        e.getMessage().split("\n", 2)[0]
```

Comment:
```suggestion
          e.getMessage().split("\n", 2)[0]
```

Nitpick: keep the same whitespace indentation